src/components/molecules/Settings/Workspace/Asset/AssetSection/index.tsx

```diff
@@ -12,23 +17,39 @@ type Asset = {
 };
 
 type Props = {
-  assets?: Asset[];
-  onCreate?: (files: FileList) => void;
-  onRemove?: (assetIds: string[]) => void;
+  assetsData: {
+    assets: AssetType[];
+    isLoading: boolean;
+    getMoreAssets: () => void;
+    createAssets: (files: FileList) => Promise<void>;
+    hasMoreAssets: boolean | undefined;
+    removeAsset: (assetIds: string[]) => Promise<void>;
+    sort?: { type?: AssetSortType | null; reverse?: boolean };
+    handleSortChange: (type?: string, reverse?: boolean) => void;
+    searchTerm?: string;
+    handleSearchTerm: (term?: string) => void;
```

Comment:
I like to put event props in the back of the type definition of props

```suggestion
    hasMoreAssets: boolean | undefined;
    sort?: { type?: AssetSortType | null; reverse?: boolean };
    searchTerm?: string;
    onRemoveAsset: (assetIds: string[]) => void;
    onSortChange: (type?: string, reverse?: boolean) => void;
    onSearchTerm: (term?: string) => void;
```